src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
+    protected String color;
```

Comment:
Since 'Figure' is an abstract class, it's okay to have instance variables like 'color'. However, if 'Figure' becomes an interface, 'color' would not be allowed as interfaces cannot have instance variables.